src/main/java/uk/gov/ch/developer/docs/security/WebSecurity.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ch.developer.docs.security;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class WebSecurity extends WebSecurityConfigurerAdapter {
+
+    @Configuration
+    @Order(1)
+    public static class TemporaryStartPageSecurityConfig extends WebSecurityConfigurerAdapter {
+        @Override
+        protected void configure(HttpSecurity http) throws Exception {
+            http.antMatcher("/signin");
+        }
+    }
+
+    @Configuration
+    @Order(2)
+    public static class DocsWebSecurityFilterConfig extends WebSecurityConfigurerAdapter {
+        @Override
+        protected void configure(HttpSecurity http) throws Exception {
+            http.antMatcher("/**");
```

Comment:
I still think things should be secure by default.

Reply:
This to be dealt with in a separate ticket, but at the moment we are leaving it like this until I am sure the functionality works.